src/scancode/cli.py

```diff
@@ -109,6 +113,13 @@
 
 notice = acknowledgment_text.strip().replace('  ', '')
 
+scan_proper_plugins = PluginManager('scan_proper')
+scan_output_plugins = PluginManager('scan_output')
+scan_proper_plugins.add_hookspecs(hookspec)
+scan_output_plugins.add_hookspecs(hookspec)
+scan_proper_plugins.load_setuptools_entrypoints('scancode_plugins')
```

Comment:
To add the command line options, the `add_cmdline_option()` method would call the `formats` plugin which in turn will return `--format` option with all the plugin formats attached.